idep/assetlist.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "idep",
+  "assets": [
+    {
+      "description": "The native token of IDEP chain",
+      "denom_units": [
+        {
+          "denom": "idep",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "idep",
+      "name": "IDEP",
+      "display": "idep",
+      "symbol": "IDEP",
+      "logo_URIs": {
+        "png": "https://github.com/IDEP-network/IDEP-Logos/raw/main/ideplogo.png"
```

Comment:
can you please point the URI to the Chain Registry `/idep/images/idep.png` ? Also, do you have one with a transparent background and centered? I fear it would look strange in dApps when cropped due to it not being centered...

Reply:
Done @JeremyParish69